mexca/video/extraction.py

```diff
@@ -2,25 +2,33 @@
 Extract facial features such as landmarks and action units.
 """
 
+from typing import Any
```

Comment:
Why do you prefer to have an "elongated" import rather than the one below?

```suggestion
from typing import Any, Dict, List, Optional, Tuple, Union
```